frontend/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,29 @@
+import { Search } from "lucide-react";
+import Input from "@mui/material/Input";
+import InputAdornment from "@mui/material/InputAdornment";
+import SearchIcon from "@mui/icons-material/Search";
+
+const Searchbar = () => {
+  return (
+    <>
```

Comment:
When you return one react element, you don't need to use fragment `<></>`. It's only needed when you need a wrapper for a list: https://react.dev/reference/react/Fragment